### DISCUSSION TITLE
Equivalent to webpack css-loader option: "modules: { auto: true }"?

### DISCUSSION THREAD

**Discussion by elzii** [parcel-bundler/parcel#5083] [General] [answered]
I'm looking to enable CSS module only for a given file extension, ex: `style.module.css` rather than it be the default project-wide. Is this possible with Parcel currently?

**Comment by mischnic** [comment, marked as answer]
That works out of the box with Parcel 2: https://v2.parceljs.org/languages/postcss/#css-modules

**Reply by elzii** [reply]
Great, thanks! Man, the note on the parcel docs about V2 coming is almost _too_ subtle...
